File: receptionHubBackend/Controllers/Dtos/RezervacijaGostDto.cs
namespace receptionHubBackend.Controllers.Dtos;

public class RezervacijaGostDto
{
    public int Id { get; set; }
    public int RezervacijaId { get; set; }
    public int GostId { get; set; }
    public string? ImeGosta { get; set; }
    public string? PrezimeGosta { get; set; }
    public string? NazivFirme { get; set; }
    public bool JeGlavniGost { get; set; }
    public string? PosebneNapomene { get; set; }
    public string? BrojRezervacije { get; set; }
}

File: receptionHubFrontend/src/types/dtos/rezervacija-gost.dto.ts
export interface RezervacijaGostDto {
  id: number;
  rezervacijaId: number;
  gostId: number;
  imeGosta?: string;
  prezimeGosta?: string;
  nazivFirme?: string;
  jeGlavniGost: boolean;
  posebneNapomene?: string;
  brojRezervacije?: string;
}

export interface KreirajRezervacijaGostDto {
  rezervacijaId: number;
  gostId: number;
  jeGlavniGost?: boolean;
  posebneNapomene?: string;
}

export interface AzurirajRezervacijaGostDto {
  jeGlavniGost?: boolean;
  posebneNapomene?: string;
}